manifests/nodejs.yml

```diff
@@ -20,6 +20,10 @@ tests/:
           TestCommandInjection:
             '*': missing_feature
             express4: v3.11.0
+        test_hardcoded_secrets.py:
+          Test_HardcodedSecrets:
+            '*': missing_feature
+            express4: v4.18.0
```

Comment:
is it not compatible with typescript and next ?

Reply:
yes, but we are not testing every iast capability in all frameworks

should next and typescript included @uurien?

Reply:
we only have implemented iast features in express4. 
We should add the tests in other apps also, but I think that it is not the goal of this PR.